torch/distributed/fsdp/_shard_utils.py

```diff
@@ -15,7 +15,6 @@
 )
 from torch.distributed._shard.sharding_spec import ShardMetadata
 from torch.distributed._tensor import DeviceMesh, DTensor, Replicate, Shard as DShard
-from torch.distributed.fsdp._debug_utils import SimpleProfiler
```

Comment:
nit: maybe you want to split this change into a separate PR? (Maybe in the future)

Reply:
This actually needs to go in the same PR as it was resulting in a circular dependency without this, since now we need to add the DTensorExtension to the FSDPstate.